textile/api-docstrings.md

```diff
@@ -620,9 +620,106 @@ Enumerates the possible types of `Message` there can be.
 | MESSAGE_CREATE | TM5 | A normal message that has been published by a user. |
 | MESSAGE_UPDATE | TM5 | An update that modifies a previously-published message (referenced by `serial`). |
 | MESSAGE_DELETE | TM5 | A deletion of a previously-published message (referenced by `serial`). |
-| META_OCCUPANCY | TM5 | Channel metadata containing information on what the current occupancy of the channel is. |
+| META | TM5 | A meta-message (a message originating from ably rather than being explicitly published on a channel), containing information such as inband channel occupancy events that has been requested by channel param. |
 | MESSAGE_SUMMARY | TM5 | A message containing the latest rolled-up summary of annotations that have been made to this message. |
 
+## class RestAnnotations
+
+
+Functionality for annotating messages with small pieces of data, such as emoji reactions, that the server will roll up into the message as a summary.
+
+| Method / Property | Parameter | Returns | Spec | Description |
+|---|---|---|---|---|
+| publish(message: Message, annotation: OutboundAnnotation) => io ||| RSAN1 | Publish a new annotation for a message. |
+|| `message` || RSAN1a1 | The message to annotate. |
+|| `annotation` || RSAN1a2 | The annotation to publish. (Must include at least the `type`. Assumed to be an annotation.create if no action is specified) |
```

Comment:
```suggestion
|| `annotation` || RSAN1a2 | The annotation to publish. (Must include at least the `type`. Assumed to be an annotation.create if no annotation `action` is specified) |
```
In case of `delete` method, there's no statement for assumption. It's supposed to be annotation.delete right